ksql-streams/src/main/java/io/confluent/ksql/execution/streams/RoutingOptions.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2020 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.execution.streams;
+
+/**
+ * These are options used for locating the host to retrieve data from.
+ */
+public interface RoutingOptions {
```

Comment:
could we consolidate other routing options like ; should route to standbys into this itself?

Reply:
This object is useful for query time options more than server options.  Maybe I can change the name?

Which other flags do you mean?  It would required adding this to many call-site parameters if I were to do that for something like KSQL_QUERY_PULL_ENABLE_STANDBY_READS.  (That has more to do with state stores than routing, per se).